### DISCUSSION TITLE
add input type file to the angular package

### DISCUSSION THREAD

**Discussion by robertIsaac** [vmware/clarity#6755] [Ideas]
add angular component similar to https://clarity.design/core-components/file/

**Comment by mathisscott** [comment]
There is a `cds-file` angular component in the core library's companion `@cds/angular` package... https://www.npmjs.com/package/@cds/angular

**Comment by robertIsaac** [comment]
@mathisscott I mean a component like that https://clarity.design/angular-components/select/
there is a big difference when working with both in the same form
